config/database.yml

```diff
@@ -15,3 +15,8 @@ development:
 test:
   <<: *default
   database: <%= ENV.fetch("DB_DATABASE", "register_trainee_teacher_data_test#{ENV['TEST_ENV_NUMBER']}") %>
+
+production:
+  <<: *default
+  variables:
+    statement_timeout: <%= ENV["STATEMENT_TIMEOUT"] || "5s" %>
```

Comment:
five seconds seems reasonably militant. how would our reports feel about it?

Reply:
I think the core issue there was around loading/iterating/transforming a large set of records in memory while blocking the request. With this though, I've tested out the threshold by exporting every trainee as a system admin and it streamed the results with no issue. Have also tried it with some ridiculously low number and it errored as expected so I think this threshold should be ok!

<img width="1055" alt="Screenshot 2023-10-16 at 13 41 03" src="https://github.com/DFE-Digital/register-trainee-teachers/assets/616080/fa8894a4-42f2-4d81-baa5-62cbc971d6d3">